bittensor/_neuron/text/advanced_server/nucleus_impl.py

```diff
@@ -87,7 +87,56 @@ def __init__(self,
         
         # -- keeps track of gradients applied
         self.backward_gradients = 0 
-        
+        self.set_fine_tuning_params()
+
+    def set_fine_tuning_params(self):
```

Comment:
Could we add return type annotations here?

Reply:
sure!